Remove cruft (#278)
ankitects/anki-manual: src/studying.md

will be buried.

Also note that a card cannot be buried and suspended at the same time. Suspending a

> buried card will unbury it whereas burying a suspended card does not work in Anki.

Reviewer: ```suggestion
buried card will unbury it. Suspended cards can't be buried.
```
